### PR TITLE
Add option to enabled/disable client stats and fix #1771

### LINKED ISSUE #1771: ProcessTiming and ItemGeometry from ClientId(0) will be skipped in TuiMonitor.item_geometry
The trait ``Monitor`` will store ``ClientStats`` of each fuzzer client in a vector, and the index is the client id ``ClientId``. So when ``SimpleEventManager`` creates ``ClientStats`` with **client id 0** in ``handle_in_broker``：
```
monitor
    .client_stats_mut_for(ClientId(0))
    .update_user_stats(name.clone(), value.clone());
```
The new ``ClientStats`` will be stored in the **0-th** position of the vector. Unfortunately, **this position will be skipped by process_timing and item_geometry**.
```
fn process_timing(&mut self) -> ProcessTiming {
        let mut total_process_timing = ProcessTiming::new();
        total_process_timing.exec_speed = self.execs_per_sec_pretty();
        if self.client_stats.len() > 1 {
            ......
            for client in self.client_stats().iter().skip(1) {
                new_path_time = client.last_corpus_time.max(new_path_time);
                new_objectives_time = client.last_objective_time.max(new_objectives_time);
            }
            ......
        }
        ......
    }
```

```
fn item_geometry(&self) -> ItemGeometry {
        let mut total_item_geometry = ItemGeometry::new();
        if self.client_stats.len() < 2 {
            return total_item_geometry;
        }
        ......
        for client in self.client_stats().iter().skip(1) {
        ......
    }
```
The result is process_timing and item_geometry is all zero in TUI.

### PR DESCRIPTION
**Pull request by domenukk**
This should fix #1771

### CODE REVIEW

File: libafl/src/monitors/tui/mod.rs [hunk 1]
```diff
@@ -494,7 +498,7 @@ impl TuiMonitor {
             .get_user_stats("edges")
             .map_or("0%".to_string(), ToString::to_string);
 
-        for client in self.client_stats().iter().skip(2) {
+        for client in self.client_stats().iter().filter(|client| client.enabled) {
```
Comment (domenukk) [inline libafl/src/monitors/tui/mod.rs:501]: I wonder why this skipped 2 instead of 1? @ToSeven any ideas? Or @andreafioraldi ?

Comment (domenukk) [reply libafl/src/monitors/tui/mod.rs]: Ah think I figured it out

File: libafl/src/monitors/disk.rs [hunk 2]
```diff
@@ -211,12 +211,11 @@ where
 
             let line = json!({
                 "run_time": current_time() - self.base.start_time(),
-                "clients": self.base.client_stats().len(),
+                "clients": self.client_stats_count(),
                 "corpus": self.base.corpus_size(),
                 "objectives": self.base.objective_size(),
                 "executions": self.base.total_execs(),
                 "exec_sec": self.base.execs_per_sec(),
-                "clients": &self.client_stats().get(1..)
```
Comment (domenukk) [inline libafl/src/monitors/disk.rs:219]: Can anyone double check if that's all right? It should never have worked in the first place..

**Comment by tokatoka** [comment]
lgtm

**Pull request merged**